packages/metrics/src/config/EnvironmentVariablesService.ts

```diff
@@ -13,12 +13,23 @@ class EnvironmentVariablesService
 {
   private namespaceVariable = 'POWERTOOLS_METRICS_NAMESPACE';
 
+  private disabledVariable = 'POWERTOOLS_METRICS_DISABLED';
```

Comment:
I know other variables are not declared as such, but since we're adding a new one and it's never reassigned, let's make it `private readonly`.